pkg/kubernetes/kubernetes.go

```diff
@@ -12,12 +12,13 @@ import (
 	"time"
 
 	"github.com/dapr/cli/pkg/print"
+	"github.com/dapr/cli/pkg/version"
 	"github.com/dapr/cli/utils"
 
 	"github.com/briandowns/spinner"
 )
 
-const daprManifestPath = "https://daprreleases.blob.core.windows.net/manifest/dapr-operator.yaml"
+var daprManifestPath string = "https://github.com/dapr/dapr/releases/download/v" + version.GetCLIVersion() + "/dapr-operator.yaml"
```

Comment:
daprManifestPath needs to use the latest version of github release. GetCLIVersion() will not return what you need.

if --runtime-version is set, you need to use this version. otherwise, you need to get the latest version of cli by using below way.

https://github.com/dapr/cli/blob/eb76cbf2200058f849814649f23212b3247a952c/pkg/standalone/standalone.go#L282

Reply:
Indeed.

Reply:
As discussed offline, we don't support --runtime-version for kubernetes mode, as of now we always pull the latest release. Made the changes.